guides/source/getting_started.md

```diff
@@ -236,7 +236,7 @@ $ bin/rails server
 ```
 
 TIP: If you are using Windows, you have to pass the scripts under the `bin`
-folder directly to the Ruby interpreter e.g. `ruby bin\rails server`.
+folder directly to the Ruby interpreter e.g. `ruby bin/rails server`.
```

Comment:
This tip is about using Windows, where the directory separator is `\`, so this was correct before.

Reply:
Fixed -- side note, have not used windows terminal in a while, will check this out on my spare laptop soon as I get time, I'm curious 🙂